docs/usage/configuration/config_documentation.md

```diff
@@ -3765,6 +3806,109 @@ redis:
   port: 6379
   password: <secret_password>
 ```
+---
+## Worker configuration for workers configuration
+These switches configure each worker them self with the a worker configuration file.
+
+Note also the configuration for the
+[main process above](#worker-configuration-for-main-process).
```

Comment:
As I suggested elsewhere: I don't think the previous batch of settings configure just communication with the main worker? I've suggested language like: they configure communication between "ordinary" and "privileged" workers.

Perhaps:

```suggestion
Note also the configuration above for
[coordinating a cluster of workers](#coordinating-workers).
```